src/Toolkit.Preview/Toolkit.Preview.WPF/UI/Controls/SignInForm/SignInForm.cs

```diff
@@ -206,7 +221,7 @@ public Uri ServerHost
         }
 
         private static readonly DependencyPropertyKey ServerHostPropertyKey =
-            DependencyProperty.RegisterReadOnly("ServerHost", typeof(Uri), typeof(SignInForm), new PropertyMetadata(null));
+            DependencyProperty.RegisterReadOnly(nameof(ServerHost), typeof(Uri), typeof(SignInForm), new PropertyMetadata(null));
```

Comment:
Right now **ServerHost** property is non-nullable, but I'm getting the impression that it _should_ be nullable.  For example, `InitializeCredentials()` below will set this to null whenever CredentialRequestInfo is also null.  And `PopulateFields()` already has null-check on this property.